packages/appstore/src/components/add-options-account/add-options-account.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Text } from '@deriv/ui';
+import { DesktopWrapper, MobileWrapper, Button } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import './add-options-account.scss';
+import { observer } from 'mobx-react-lite';
+import { useStores } from 'Stores';
+
+const AddOptionsAccount: React.FC = () => {
+    const { ui } = useStores();
+    return (
+        <React.Fragment>
+            <div className='add-options-account'>
+                <DesktopWrapper>
+                    <div className='add-options-account__title'>
+                        <Text type='paragraph-2' bold>
```

Comment:
- What is `type` prop?
- What is `bold` supposed to do?

Reply:
I thought we need to use this if we using deriv/ui ?

Reply:
I see, we will need to discuss this further with the whole team since we're going to have 2 different versions of components from Deriv-app and Deriv-ui.